_posts/2024-09-06-this-week-in-rails.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "Solid Cache and Queue as default backends and more"
+categories: news
+author: Wojtek
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2024-09-06
+---
+
+Hi, [Wojtek](https://x.com/morgoth85) here. Let's explore this week's changes in the Rails codebase.
+
+[Add Solid Cache](https://github.com/rails/rails/pull/52790)  
+Solid Cache will be the new default caching backend for production deployments out of the box in Rails 8.
+
+[Add Solid Queue](https://github.com/rails/rails/pull/52804)  
+Configure Solid Queue as the default Active Job backend alongside Solid Cache. Both can be skipped with `--skip-solid`.
+
+[Allow registering test directories for code statistics](https://github.com/rails/rails/pull/52747)  
+Make it easier for third party gems, to register test directories.
+
+[Silence healthcheck requests from the log](https://github.com/rails/rails/pull/52789)  
+Add `Rails::Rack::SilenceRequest` middleware and use it via `config.silence_healthcheck_path = path` to silence requests to "/up".
+This prevents the Kamal-required healthchecks from clogging up the production logs.
+
+[Fix authentication generator double signature](https://github.com/rails/rails/pull/52786)  
+Previously the session id stored in the cookies was signed twice: with `cookies.signed` and `session.signed_id`.
```

Comment:
Was wrongly signed?

Reply:
@vipulnsward What do you mean exactly?
It was correctly signed before, but twice which was unnecessary